pipeline/test_transform.py

```diff
@@ -78,7 +78,7 @@ def test_normalize_datetimes():
     df = normalize_datetimes(df)
 
     assert df.to_dict() == {'Plant Name': {0: 'PlantA'},
-                            'Last Watered': {0: pd.Timestamp('2023-12-18 14:03:04+0000', tz='GMT')},
+                            'Last Watered': {0: pd.Timestamp('2023-12-18 14:03:04')},
```

Comment:
Does the inclusion of a timezone work for recording taken (as you didn't remove it from that line). Is this because the data comes in slightly differently for the times?

Reply:
Yeah one time already comes in as a timestamp the other is different